ash/src/extensions/ext/debug_marker.rs

```diff
@@ -11,7 +11,7 @@ pub struct DebugMarker {
 }
 
 impl DebugMarker {
-    pub fn new<I: InstanceV1_0, D: DeviceV1_0>(instance: &I, device: &D) -> DebugMarker {
+    pub fn new<I: InstanceV1_0, D: DeviceV1_0>(instance: &I, device: &D) -> Self {
```

Comment:
Thanks for these, but we should probably do them in a separate PR with `clippy::use_self` globally disabled.

Reply:
Addendum: Also notice that you forgot to remove these from the struct constructor just below, last expression of these functions.